### PR TITLE
[jss-plugin-expand]Added the ability to expand the attributes for border-bottom

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**
__What would you like to add/fix?__ 
Expanding border-bottom which is referred in issue #1023 

__Corresponding issue (if exists):__

#1023

### CODE REVIEW

**Comment by JayaKrishnaNamburu** [comment]
Hey @HenriBeck i tested this in my local. Please let me know if i should change for other attributes and well and if need to update any docs 😃

**Comment by HenriBeck** [comment]
@JayaKrishnaNamburu can you please also add the objects for `border-top`, `border-left` and `border-right`?

**Comment by JayaKrishnaNamburu** [comment]
Sure, will do them and update you :+1:

**Comment by HenriBeck** [comment]
Also please add a changelog entry under bug fixes.
You will also need to add the following on top of the changelog
```
## Next

### Bug fixes
```

**Comment by JayaKrishnaNamburu** [comment]
Hey @HenriBeck please check now, i have updated the PR.

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Next
+
+### Bug fixes
+
+- [jss-plugin-expand] Fix attributes spread for `border-bottom`, `border-top`, `border-left` and `border-right`
```
Comment (HenriBeck) [inline changelog.md]: Please also link the PR here

Comment (JayaKrishnaNamburu) [reply changelog.md]: Done 👍

**Comment by JayaKrishnaNamburu** [comment]
Added PR link to the changelog

File: packages/jss-plugin-expand/src/props.js [hunk 2]
```diff
@@ -141,6 +141,26 @@ export const customPropObj = {
     style: 'border-style',
     color: 'border-color'
   },
+  'border-bottom': {
+    width: 'border-bottom-width',
+    style: 'border-bottom-style',
+    color: 'border-bottom-color'
+  },
+  'border-top': {
+    width: 'border-bottom-width',
```
Comment (HenriBeck) [inline packages/jss-plugin-expand/src/props.js]: These need to be `border-top` etc.

Comment (JayaKrishnaNamburu) [reply packages/jss-plugin-expand/src/props.js]: Sorry for that, i am updating

**Comment by HenriBeck** [comment]
Thanks for the PR 👍

**Pull request merged**